src/pages/_app.tsx

```diff
@@ -90,8 +89,8 @@ const SafeWebCore = ({ Component, pageProps }: AppProps): ReactElement | null =>
 
         <Notifications />
       </AppProviders>
-    </Provider>
+    </>
   )
 }
 
-export default SafeWebCore
+export default storeHydrator(SafeWebCore)
```

Comment:
Any specific reason it needs to wrap the entire app as opposed to just calling it from InitApp or in AppProviders?

Reply:
It has to call the hydration action before any other actions are dispatched.